config/routes.rb

```diff
@@ -2,9 +2,11 @@
   scope "(:locale)", locale: /en|vi/ do
     get "/login", to: "sessions#new"
     get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
     get "/about", to: "static_pages#about"
     get "/contact", to: "static_pages#contact"
     get "/service", to: "static_pages#service"
     root "static_pages#home"
+    resources :users, only: %i(show update edit new index)
```

Comment:
If there is more than 4 action, use `except`

Reply:
I will using resouces :user for all method in Userscontroller